.ci/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:${NODE_JS_VERSION}
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+COPY package*.json ./
```

Comment:
So there's an issue here with this line, and there's a couple ways we can solve it. 

1. move the `docker-compose.yml` up one directory to the root of the repo. 
2. Adjust the docker-compose build context to reference the correct location. 

the problem right now is that `Dockerfile` is trying to copy a file that doesn't exist. (it exists one directory up).

Reply:
I went for option 2 :)